pysipp/report.py

```diff
@@ -46,12 +46,22 @@ def emit_logfiles(agents2procs, level='warn', max_lines=100):
     """
     emit = getattr(log, level)
     for ua, proc in agents2procs:
+        i=5
```

Comment:
For one I'd just use a `range(5)` here instead of the `while` loop.
@fchareyron Can you explain how this helps?
As far as I remember `stderr` capture should be synchronous so it shouldn't make any difference how long you wait? Maybe you found something else in your use?